locals.tf

```diff
@@ -3,4 +3,5 @@ locals {
   step_function_name           = "${local.resource_prefix}-parse-judgment"
   lambda_name_parse_judgment   = "${local.resource_prefix}-parse-judgment-lambda"
   lambda_name_trigger          = "${local.resource_prefix}-parse-judgment-trigger"
+  tre_data_bucket              = "arn:aws:s3:::${var.tre_data_bucket}"
```

Comment:
I think we should directly reference it instead of using locals to keep it consistent